src/components/common/MessageComposer.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react';
+import * as C from '../../constants/constants';
+
+class MessageComposer extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+
+    this.state = {
+      text: ''
+    };
+  }
+
+  handleSubmit(event) {
+    const textContent = event.target.value.trim();
```

Comment:
Move this line inside the `if` block. The `textContent` It's not used outside the `if` block.